packages/@aws-cdk/assert/lib/test-stack.ts

```diff
@@ -0,0 +1,7 @@
+import cdk = require('@aws-cdk/cdk');
+
+export class TestStack extends cdk.Stack {
```

Comment:
1. Is this a good idea to put in a reusable place?
1. I started to find all the places we *should* use this, but I missed a few and now I'm questioning if this is too much to include here?

Reply:
I think this class is no longer needed but the test clean up seems like it was a good idea?